faq.html

```diff
@@ -0,0 +1,155 @@
+                </h6>
+            </div>
+            <h3>How can I get Vita3K Android?</h3>
+            <div class="my-5">
+                <h6>
+                    There is <strong>No Vita3K version for Android</strong> at the moment. Any links you find for .apk's are FAKE. Android version is under development.
+                    There is no ETA on this, and there will be many bugs and it will take a while until it's ready to begin porting. Again, there is <strong>NO Vita3K Android</strong>.
+                </h6>
+            </div>
+            <h3>Does Vita3K support commercial games?</h3>
+            <div class="my-5">
+                <h6>
+                    <strong>Some</strong>. The emulator is in an early stage of development and a lot of features required to run many commercial games haven't been implemented yet. Currently, many games made with GameMaker Engine work flawlessly.
+                    More games will be supported when the hardware/firmware emulation is accurate enough to run them. It all depends on the amount of developers working on this project, their skills, and free time, etc. Ignore any estimates and guesses you find out there.
+                </h6>
+            </div>
+            <h3>How to get games?</h3>
+            <div class="my-5">
+                <h6>
+                    For homebrew, get them via <a href="https://vitadb.rinnegatamante.it/">VitaDB</a>
+                    For commercial games, dump them with <a href="https://github.com/TheOfficialFloW/NoNpDrm/releases">NoNpDrm</a>. If the game is known to not work with NoNpDrm, you can optionally replace the executable with a <a href="https://github.com/CelesteBlue-dev/PSVita-RE-tools/tree/master/FAGDec/build">FAGDec</a> dump. Vitamin dumps are <strong>not</strong> supported.
+                </h6>
+            </div>
+            <h3>How to run games?</h3>
+            <div class="my-5">
+                <h6>First you have to install them. When you do, you only need to run the emulator and it will display a <em>Game Select</em> Screen.</h6>
+            </div>
+            <h3>How to install games?</h3>
+            <div class="my-5">
+                <h6>
+                    Windows:<br />
+                    Right click on a <code>.vpk</code>, go to Properties and click "Change..." to associate <code>Vita3K.exe</code> with <code>.vpk</code> files. Double-clicking them should work after that.
+                    Alternatively, drag and drop games to <code>Vita3K.exe</code>.<br /><br />
+                    Windows/Linux/macOS:<br />
+                    Games can be installed via the emulator directly through the <code>File > Install .pkg</code> or <code>File > Install .zip .vpk</code>.<br />
+                    Supply them as a command-line argument to the executable (the latter is necessary for Linux/Mac OS).
```

Comment:
I don't believe it's necessary to supply the game as a command-line argument for linux/mac. The command-line argument option should be phrased as an alternative to the GUI installation method.